torchvision/models/shufflenetv2.py

```diff
@@ -223,11 +223,37 @@ class ShuffleNet_V2_X1_0_Weights(WeightsEnum):
 
 
 class ShuffleNet_V2_X1_5_Weights(WeightsEnum):
-    pass
+    IMAGENET1K_V1 = Weights(
+        url="https://download.pytorch.org/models/shufflenetv2_x1_5-3c479a10.pth",
+        transforms=partial(ImageClassification, crop_size=224),
```

Comment:
In the recipe you mentioned that you used `--val-resize-size=232` but that's not included here. Can you please add it and provide execution runs with the pre-trained weights that confirm the batch-size=1/gpu=1 accuracies?

Reply:
Thanks for noticing this! I forgot to add this and as a result the accuracy went down a bit for the eval. I have added this and update the resulting eval as well. The test execution run is on the PR description.